tests/fuzz/fuzz_common.h

```diff
@@ -0,0 +1,62 @@
+//
+// Created by profanter on 18.07.17.
+// Copyright (c) 2017 fortiss GmbH. All rights reserved.
+//
+
+#ifndef OPEN62541_FUZZ_COMMON_H_H
```

Comment:
Yes, within open62541 this wouldn't be a problem, but there is the magic build script here:
https://github.com/google/oss-fuzz/blob/master/projects/open62541/build.sh

So for building the .cc file, we also need to separately build the testing_networklayer.c file. I had this in a test before, but it made the build script more blown up...